stripe/connectCallback.js

```diff
@@ -0,0 +1,102 @@
+import Stripe from 'stripe';
+import { failure, redirect, redirectWithError } from '../libs/response-lib';
+import * as dynamoDbLib from '../libs/dynamodb-lib';
+
+export const main = async event => {
+  /** @type {string} code - authorization code returned from Stripe */
+  const code = event.queryStringParameters.code;
+  const stripeConnectToken = event.queryStringParameters.state;
+  const params = {
+    TableName: process.env.COMPANIES_TABLE_NAME,
+    ExpressionAttributeValues: {
+      ':stripeConnectToken': stripeConnectToken
+    },
+    FilterExpression: 'stripeConnectToken = :stripeConnectToken'
+  };
+
+  let company;
+
+  try {
+    const response = await dynamoDbLib.call('scan', params);
+
+    if (response.Count === 0) {
+      const error = 'resource_not_found';
+      const errorMsg = 'The company does not exist in the DB';
+      return redirectWithError(process.env.APPLICATION_URL, error, errorMsg);
```

Comment:
should we log some info to track such failed requests?

Reply:
I suppose if AWS somehow logs all the requests with all the query params and other params, it would be good enough